### PR TITLE
feat: airplay

### PR DESCRIPTION
**Pull request by dan-ziv**

### CODE REVIEW

File: src/components/AirPlayButton.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    this.props.addPlayerClass(style.casting);
+  };
+
+  airPlayEndedHandler = () => {
+    this.setState({isActive: false});
+    this.props.removePlayerClass(style.casting);
+  };
+
+  render(props: Object) {
+    if (!this.state.isAvailable) return undefined;
```
Comment (OrenMe) [inline src/components/AirPlayButton.js:49]: curly brackets even for single line if's.
Don't we have lint rule for it?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          api_key: $NPM_TOKEN
+          email: $NPM_EMAIL
+          skip_cleanup: true
+          on:
+            tags: true
+            branch: master
+      after_deploy:
+        - currentVersion=$(npx -c 'echo "$npm_package_version"')
+        - chmod +x ./scripts/after_deploy.sh
+        - ./scripts/after_deploy.sh "ima" "$currentVersion" "$JENKINS_TAG_TOKEN"
```
Comment (Yuvalke) [inline .travis.yml:58]: change `ima` to `airplay`

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        skip_cleanup: true
+        tag: canary
+        on:
+          tags: false
+          branch: master
+      after_deploy:
+        - currentVersion=$(npx -c 'echo "$npm_package_version"')
+        - echo $currentVersion
+        - chmod +x ./scripts/after_deploy.sh
+        - ./scripts/after_deploy.sh "ima" "$currentVersion" "$JENKINS_CANARY_TOKEN"
```
Comment (Yuvalke) [inline .travis.yml:76]: same as mentioned above

File: package.json [hunk 4]
```diff
@@ -0,0 +1,120 @@
+{
+  "name": "@playkit-js/playkit-js-airplay",
+  "version": "0.0.1",
```
Comment (Yuvalke) [inline package.json:3]: yarn init suppose to start from 1.0.0 on the latest version

File: karma.conf.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  }
+};
+
+module.exports = function (config) {
+  let karmaConf = {
+    logLevel: config.LOG_INFO,
+    browserDisconnectTimeout: 30000,
+    browserNoActivityTimeout: 60000,
+    customLaunchers: launchers,
+    browsers: ['Chrome_browser', 'Firefox'],
```
Comment (Yuvalke) [inline karma.conf.js:30]: I think you can set Safari as a browser and add the relevant tests [karma safari luncher](https://www.npmjs.com/package/karma-safari-launcher)

**Review by Yuvalke — CHANGES_REQUESTED** [review]

File: .babelrc [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  "ignore": ["node_modules/**/*"],
+  "plugins": [
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-transform-flow-strip-types",
+    "@babel/plugin-proposal-export-namespace-from",
+    "@babel/plugin-proposal-export-default-from",
+    "@babel/plugin-transform-property-mutators",
+    "@babel/plugin-proposal-object-rest-spread",
+    [
+      "@babel/plugin-transform-spread",
```
Comment (Yuvalke) [inline .babelrc:16]: what is that one for? it does not exist on the other plugins or UI

File: test/setup/load-specs.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+/**
```
Comment (Yuvalke) [inline test/setup/load-specs.js:1]: would you add tests after?

**Review by Yuvalke — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+<!-- If you are raising a bug playing a stream, you must fill out the following or your issue may not be responded to. For features or improvements, you may delete this. -->
+
+##### Prerequisites
+
+- [ ] Have you checked for duplicate [issues](https://github.com/kaltura/playkit-js-ima/issues): **\_\_**
```
Comment (OrenMe) [inline .github/ISSUE_TEMPLATE.md:5]:
```suggestion
- [ ] Have you checked for duplicate [issues](https://github.com/kaltura/playkit-js-airplay/issues): **\_\_**
```

File: karma.conf.js [hunk 9]
```diff
@@ -0,0 +1,65 @@
+let webpackConfig = require('./webpack.config.js');
+//Need to remove externals otherwise they won't be included in test
+delete webpackConfig.externals;
+// Need to define inline source maps when using karma
+webpackConfig.devtool = 'inline-source-map';
```
Comment (OrenMe) [inline karma.conf.js:5]: Use the new convention from other repos for webpack config

File: package.json (hunk 4 above)
Comment (OrenMe) [inline package.json:3]: set this to 1.0.0 for the official release

File: karma.conf.js (hunk 9 above)
Comment (OrenMe) [reply karma.conf.js:5]: https://github.com/kaltura/playkit-js/pull/479/

File: src/components/AirPlayButton.js [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+    this.setState({isActive: false});
+    this.props.removePlayerClass(style.casting);
+  };
+
+  render(props: Object) {
+    if (!this.state.isAvailable) {
+      return undefined;
+    }
+    return (
+      <div role="button" className={style.controlButtonContainer}>
```
Comment (OrenMe) [inline src/components/AirPlayButton.js:57]: Needs tooltip.
Why use div with button role?
Why not use something like the fullscreen button:

https://github.com/kaltura/playkit-js-ui/blob/1019e6e5ddafa2e1019ab4f40dc10c88c80a70d6/src/components/fullscreen/fullscreen.js#L115-L131

**Review by OrenMe — CHANGES_REQUESTED** [review]

File: .babelrc (hunk 6 above)
Comment (dan-ziv) [reply .babelrc:16]: removed

File: karma.conf.js (hunk 9 above)
Comment (dan-ziv) [reply karma.conf.js:5]: done

**Review by OrenMe — DISMISSED** [review]

**Review by OrenMe — APPROVED** [review]

**Pull request merged**